app/controllers/search_controller.rb

```diff
@@ -3,5 +3,7 @@ def index
     @query = Activity.ransack(params[:q])
     @activities = @query.result(distinct: true)
     @users = User.all
+    skip_policy_scope
+    skip_authorization
```

Comment:
Skip authorization & policy here so that any user/visitor can use the search bar.